manticore/core/cpu/abstractcpu.py

```diff
@@ -605,7 +724,15 @@ def render_instruction(self):
 
     def render_register(self, reg_name):
         result = ""
-        value = self.read_register(reg_name)
+
+        self.will_read_register.disable()
```

Comment:
That's it I'll remove the cli/sti from there. Lets document it and force the user to do it explicitly for now.  I'm not really fixing anything fully with disabling it just here.